build/boilerplate.go.txt

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
What's this file for?

Reply:
This is header for auto-generated files, as method moves from vendor/k8s.io/code-generator/generate-groups.sh invoke to the direct go run k8s.io/code-generator/cmd/client-gen call